platform/resource-bundle-locator/src/main/java/ddf/platform/resource/bundle/locator/ResourceBundleLocatorImpl.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.platform.resource.bundle.locator;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+import java.util.Locale;
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
+
+public class ResourceBundleLocatorImpl implements ResourceBundleLocator {
+
+  private File resourceBundleBaseDir;
+
+  @Override
+  public ResourceBundle getBundle(String baseName) {
+    return getBundle(baseName, Locale.getDefault());
+  }
+
+  @Override
+  public ResourceBundle getBundle(String baseName, Locale locale) {
+    File resourceBundleDir = resourceBundleBaseDir;
+    if (resourceBundleDir == null) {
+      resourceBundleDir = new File(String.format("%s/etc/i18n/", System.getProperty("ddf.home")));
+    }
+
+    try {
+      URL[] urls = {resourceBundleDir.toURI().toURL()};
+
+      try (URLClassLoader loader = new URLClassLoader(urls)) {
+        return AccessController.doPrivileged(
+            (PrivilegedAction<ResourceBundle>)
+                () -> ResourceBundle.getBundle(baseName, locale, loader));
+      }
+    } catch (IOException e) {
+      throw new MissingResourceException(
+          "An error occurred while creating class loader to URL for ResourceBundle: "
+              + baseName
+              + ","
+              + locale.getCountry()
+              + ","
+              + locale.getLanguage(),
```

Comment:
If you are going to put all locales parts than make sure to put all of them. The fact is that resource bundles loads not just language and country but variants as well. It is better to simply do `+ locale`